components/buttons/buttons.hbs

```diff
@@ -0,0 +1,481 @@
+<div class="container">
+  <div class="row">
+    <div class="col-12">
+      <br /><br />
+      <h1>Milestone buttons</h1>
+      <h2>All type of buttons</h2>
+      <h3>Without Icons</h3>
+      <div class="msds-btn-group--left">
```

Comment:
Its not valid BEM just to add a modifier class to an html element, you must apply the bem element and then add the modifier on top of it